panda/src/tinydisplay/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+set(P3TINYDISPLAY_HEADERS
+        config_tinydisplay.h
+        tinyGeomMunger.I tinyGeomMunger.h
+        tinySDLGraphicsPipe.I tinySDLGraphicsPipe.h
+        tinySDLGraphicsWindow.I tinySDLGraphicsWindow.h
+        tinyGraphicsBuffer.I tinyGraphicsBuffer.h
+        tinyGraphicsStateGuardian.I tinyGraphicsStateGuardian.h
+        tinyTextureContext.I tinyTextureContext.h
+        tinyWinGraphicsPipe.I tinyWinGraphicsPipe.h
+        tinyWinGraphicsWindow.I tinyWinGraphicsWindow.h
+        tinyXGraphicsPipe.I tinyXGraphicsPipe.h
+        tinyXGraphicsWindow.I tinyXGraphicsWindow.h
+        tinyOffscreenGraphicsPipe.I tinyOffscreenGraphicsPipe.h
+        zbuffer.h
+        zfeatures.h
+        zgl.h
+        zline.h
+        zmath.h
+        ztriangle.h
+        ztriangle_two.h
+        ztriangle_code_1.h
+        ztriangle_code_2.h
+        ztriangle_code_3.h
+        ztriangle_code_4.h
+        ztriangle_table.h
+        store_pixel.h
+        store_pixel_code.h
+        store_pixel_table.h
+)
+
+set(P3TINYDISPLAY_SOURCES
+        clip.cxx
+        config_tinydisplay.cxx
+        error.cxx
+        image_util.cxx
+        init.cxx
+        td_light.cxx
+        memory.cxx
+        specbuf.cxx
+        store_pixel.cxx
+        td_texture.cxx
+        tinyGeomMunger.cxx
+        tinyGraphicsBuffer.cxx
+        tinyGraphicsStateGuardian.cxx
+        tinyOffscreenGraphicsPipe.cxx
+        tinySDLGraphicsPipe.cxx
+        tinySDLGraphicsWindow.cxx
+        tinyTextureContext.cxx
+        tinyWinGraphicsPipe.cxx
+        tinyWinGraphicsWindow.cxx
+        tinyXGraphicsPipe.cxx
+        tinyXGraphicsWindow.cxx
+        vertex.cxx
+        zbuffer.cxx
+        zdither.cxx
+        zline.cxx
+        zmath.cxx
```

Comment:
maybe missing 
  ztriangle_table.cxx
  ztriangle_1.cxx
  ztriangle_2.cxx
  ztriangle_3.cxx
  ztriangle_4.cxx
  srgb_tables.cxx

some these symbols defined in there are not found when linking static

Reply:
Yes, they are missing, and they should also be marked as not participating in the unity build system due to their size.